### PR TITLE
Logo & Favicons

### PR DESCRIPTION
**Pull request by chris-short**
- Added multiple formats of the logo to /img
- Added full compliment for favicons
- head.html edited accordingly

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: assets/img/favicon/manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+ "name": "App",
+ "icons": [
+  {
+   "src": "\/android-icon-36x36.png",
```
Comment (stockholmux) [inline assets/img/favicon/manifest.json:5]: the `\/andorid....` looks weird. Should it be escaped like that?

Comment (chris-short) [reply assets/img/favicon/manifest.json:5]: I've seen it both ways. But, these paths aren't accurate either. I'll need to edit this regardless.

Comment (chris-short) [reply assets/img/favicon/manifest.json]: Pushed an update

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Build the Ruby image from source
+FROM ruby:3.0 AS ruby-build
```
Comment (stockholmux) [inline Dockerfile:2]: Can you contribute the Dockerfile under a different PR?

**Review by stockholmux — CHANGES_REQUESTED** [review]
Favicons seem to be working, but the Dockerfile should go elsewhere

File: Dockerfile (hunk 2 above)
Comment (chris-short) [reply Dockerfile:2]: That wasn't supposed to be here...  
I loathe git

Comment (chris-short) [reply Dockerfile]: Removed from this PR.

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**